src/scenes/Scene.js

```diff
@@ -2,13 +2,13 @@
  * @author mrdoob / http://mrdoob.com/
  */
 
-THREE.Scene = function () {
+THREE.Scene = function (fog) {
```

Comment:
@WestLangley Definitely inclined to agree with you. I only went with this approach because it's what ```THREE.HemisphereLight``` does:

https://github.com/mrdoob/three.js/blob/dev/src/loaders/ObjectLoader.js#L549
https://github.com/mrdoob/three.js/blob/dev/src/lights/HemisphereLight.js#L15

However, I guess in the case of ```THREE.HemisphereLight``` the class is a bit more specialised i.e. not likely to become littered with arbitrarily ordered optional parameters.

@mrdoob Your feedback on the matter would be appreciated.